src/script/E2EIdentity/E2EIdentityEnrollment.ts

```diff
@@ -358,12 +374,14 @@ export class E2EIHandler extends TypedEventEmitter<Events> {
         hideClose: true,
         hideSecondary: !snoozable,
         primaryActionFn: async () => {
-          await this.enroll(snoozable);
+          await this.enroll({snoozable});
           resolve();
         },
         secondaryActionFn: async () => {
           const delay = await this.startTimers();
-          this.showSnoozeConfirmationModal(delay);
+          if (delay > 0) {
+            this.showSnoozeConfirmationModal(delay);
+          }
```

Comment:
We don't want to see the "you can update your cert in 0 (or -1) seconds in the device settings" modal.